src/routes/console/project-[project]/storage/bucket-[bucket]/settings/+page.svelte

```diff
@@ -69,78 +69,72 @@
         } else isExtensionsDisabled = true;
     }
 
-    async function toggleBucket() {
-        try {
-            await sdkForProject.storage.updateBucket(
-                $bucket.$id,
-                $bucket.name,
-                undefined,
-                undefined,
-                enabled
-            );
-            invalidate(Dependencies.BUCKET);
-            addNotification({
-                message: `${$bucket.name} has been updated`,
-                type: 'success'
-            });
-            trackEvent('submit_bucket_enable', {
-                value: enabled
-            });
-        } catch (error) {
-            addNotification({
-                message: error.message,
-                type: 'error'
-            });
-        }
+    interface IUpdateBucket {
+        id: string;
+        name: string;
+        permissions?: string[];
+        fileSecurity?: boolean;
+        enabled?: boolean;
+        maximumFileSize?: number;
+        allowedFileExtensions?: string[];
+        compression?: string;
+        encryption?: boolean;
+        antivirus?: boolean;
     }
-    async function updateName() {
-        try {
-            await sdkForProject.storage.updateBucket($bucket.$id, bucketName);
-            invalidate(Dependencies.BUCKET);
-            addNotification({
-                message: 'Name has been updated',
-                type: 'success'
-            });
-            trackEvent('submit_bucket_update_name');
-        } catch (error) {
-            addNotification({
-                message: error.message,
-                type: 'error'
-            });
-        }
+
+    interface IUpdateBucketMisc {
```

Comment:
it seems like we use `type` rather than `interface` across the codebase so it might make more sense to use `type` here for consistency.

Reply:
done